packages/flutter_tools/lib/src/usage.dart

```diff
@@ -54,7 +56,9 @@ const String reloadExceptionTargetPlatform = 'cd27';
 const String reloadExceptionSdkName = 'cd28';
 const String reloadExceptionEmulator = 'cd29';
 const String reloadExceptionFullRestart = 'cd30';
-// Next ID: cd31
+
+const String enabledFlutterFeatures = 'cd31';
```

Comment:
If you could take cd32 for this then I'll use cd31 in https://github.com/flutter/flutter/pull/36208

Reply:
Done, next time we'll rowshambo for it :)